### PR TITLE
curly -> straight apostrophes

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Adding to #16324

### CODE REVIEW

File: files/en-us/_redirects.txt [hunk 1]
```diff
@@ -11659,7 +11659,7 @@
 /en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs	/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs
 /en-US/docs/Web/HTTP/Basics_of_HTTP/Introduction_to_www_and_non-www_URLs	/en-US/docs/Web/HTTP/Basics_of_HTTP/Choosing_between_www_and_non-www_URLs
 /en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Complete_list_of_MIME_types	/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
-/en-US/docs/Web/HTTP/CORS/Errors/Reason:_CORS_header_‘Origin’_cannot_be_added	/en-US/docs/Web/HTTP/CORS/Errors/CORSOriginHeaderNotAdded
+/en-US/docs/Web/HTTP/CORS/Errors/Reason:_CORS_header_‘Origin'_cannot_be_added	/en-US/docs/Web/HTTP/CORS/Errors/CORSOriginHeaderNotAdded
```
Comment (teoli2003) [inline files/en-us/_redirects.txt:11662]: Please don't run scripts on _redirects.txt.
```suggestion
/en-US/docs/Web/HTTP/CORS/Errors/Reason:_CORS_header_‘Origin’_cannot_be_added	/en-US/docs/Web/HTTP/CORS/Errors/CORSOriginHeaderNotAdded
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Impressive how they creep back in… On fix and we can merge this.

**Comment by OnkarRuikar** [comment]
GitHub is bugging out on the redirects file. Had to do push it from local.

> Impressive how they creep back in…

They are also known as `typographer's apostrophes`. Either people have configured their keyboards or some locale has them by default.

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
> GitHub is bugging out on the redirects file. Had to push it from local.
> 
> > Impressive how they creep back in…
> 
> They are also known as `typographer's apostrophes`. Either people have configured their keyboards or some locale has them by default.
I know them, but I've cleaned them 2 weeks ago :-)

> 
> We need to add this rule to the linter WDYT?

Yeah, that would be good (also the curly quotes)

**Pull request merged**